pkg/artifacts/signable.go

```diff
@@ -217,6 +217,10 @@ func ExtractOCIImagesFromResults(obj objects.TektonObject, logger *zap.SugaredLo
 		if key.Name != "IMAGES" {
 			continue
 		}
+		if key.Value.StringVal == "" {
```

Comment:
Can we add some tests to signable_test.go to verify behavior for the different types of result formats?

I suspect we might not even need this check at all because of the `if trimmed == 
""` below, but it would be great to have tests to confirm.

Reply:
Good point! I added the tests and removed this check